### PR TITLE
[inspector] Add activity features

### PR DESCRIPTION
**Pull request by jasonLaster**
I think you guys are going to like this one. I sat down this afternoon and added the context field to the activity tree: (photo)

Now, one thing you can't see in the diff is that I taped this work as I went through it... So, the screencast will be made available sometime tonight when it finishes exporting / uploading to youtube. It's a little bit long, ya'll know that I tend to get excited so you'll hear me say I just want to do one small thing and ofcourse feature creep creeps in... Anyways, I'm really happy with the basics of this feature - you can see all of the sub tasks below but a lot was added to make the activity tree super interesting. 

### Features:
+ Add context to activity tree
+ Clean up styles so that context and event are links
+ Wire up context so that it takes you to the view
+ Add Model events to the activity pane
+ add View Model isDetatche helper
+ add View Model getName helper

### CODE REVIEW

File: extension/css/inspector/partials/_tree-view.scss [hunk 1]
```diff
@@ -40,7 +40,7 @@ li.tree-group-item {
 
   &:hover {
     color: #428BCA;
-    cursor: pointer;
+    // cursor: pointer;
```
Comment (samccone) [inline extension/css/inspector/partials/_tree-view.scss]: ha why not :x:

Comment (jasonLaster) [inline extension/css/inspector/partials/_tree-view.scss]: cause... 
actually - not doing this lets the other links pop

Comment (samccone) [inline extension/css/inspector/partials/_tree-view.scss]: kk

**Comment by jasonLaster** [comment]
In!

**Pull request closed without merging**